CHANGELOG.md

```diff
@@ -45,7 +45,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   materialized in the working copy. The default option ("diff") renders
   conflicts as a snapshot with a list of diffs to apply to the snapshot.
   The new "snapshot" option just renders conflicts as a series of snapshots,
-  showing each side of the conflict and the base(s).
+  showing each side and base of the conflict. The new "git-diff3" option
+  replicates Git's "diff3" conflict style, meaning it is more likely to work
+  with external tools, but it doesn't support conflicts with more than 2 sides.
```

Comment:
We could use the ancestor line for each base and the separator line for each side. I don't know if that would be more or less surprising for users and tools.

Reply:
I was thinking it would be easier to implement the basic version first, and then if support for more than 2 sides is required, it could be added in a later version. One reason I think the current behavior might be more desirable is that it seems very difficult to manually resolve many-sided conflicts just by looking at the sides/bases, so it seems like providing diffs in that case would be more helpful, but I'm not sure.